src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -191,50 +191,50 @@ limitations under the License.
-    <attribute side="server" code="0x000F" define="OPTIONS" type="BITMAP8" min="0x00" max="0x03" writable="true" default="0x00" optional="false" introducedIn="l&amp;o-1.0-15-0014-04">options</attribute>
-    <attribute side="server" code="0x4000" define="START_UP_CURRENT_LEVEL" type="INT8U" isNullable="true" writable="true" optional="true" introducedIn="l&amp;o-1.0-15-0014-04">
-      <description>start up current level</description>
+    <attribute side="server" code="0x0010" define="ON_OFF_TRANSITION_TIME"        type="INT16U"                             writable="true"  default="0x0000" optional="true">OnOffTransitionTime</attribute>
+    <attribute side="server" code="0x0011" define="ON_LEVEL"                      type="INT8U"    isNullable="true"         writable="true"                   optional="false">OnLevel</attribute>
+    <attribute side="server" code="0x0012" define="ON_TRANSITION_TIME"            type="INT16U"   isNullable="true"         writable="true"                   optional="true">OnTransitionTime</attribute>
+    <attribute side="server" code="0x0013" define="OFF_TRANSITION_TIME"           type="INT16U"   isNullable="true"         writable="true"                   optional="true">OffTransitionTime</attribute>
+    <attribute side="server" code="0x0014" define="DEFAULT_MOVE_RATE"             type="INT8U"    isNullable="true"         writable="true"                   optional="true">DefaultMoveRate</attribute>
+    <attribute side="server" code="0x000F" define="OPTIONS"                       type="BITMAP8"  min="0x00" max="0x03"     writable="true"  default="0x00"   optional="false">Options</attribute>
+    <attribute side="server" code="0x4000" define="START_UP_CURRENT_LEVEL"        type="INT8U"    isNullable="true"         writable="true"                   optional="true">
+      <description>StartUpCurrentLevel</description>
       <access op="read" role="view"/>
       <access op="write" role="manage"/>
     </attribute>
     <command source="client" code="0x00" name="MoveToLevel" optional="false" cli="zcl level-control mv-to-level">
       <description>
         Command description for MoveToLevel
       </description>
-      <arg name="level" type="INT8U"/>
-      <arg name="transitionTime" type="INT16U"/>
-      <arg name="OptionsMask" type="BITMAP8"/>
+      <arg name="Level"           type="INT8U"/>
+      <arg name="TransitionTime"  type="INT16U" isNullable="true"/>
+      <arg name="OptionsMask"     type="BITMAP8"/>
       <arg name="OptionsOverride" type="BITMAP8"/>
     </command>
     <command source="client" code="0x01" name="Move" optional="false" cli="zcl level-control move">
       <description>
         Command description for Move
       </description>
-      <arg name="moveMode" type="MoveMode"/>
-      <arg name="rate" type="INT8U"/>
+      <arg name="MoveMode" type="ENUM8"/>
```

Comment:
Why the type change here?  Because the name collision was leading to code that was not compiling?

I think the usual way we have dealt with that was to rename the enum to someting like MoveModeEnum.  Please don't use ENUM8, which causes us to use non-typesafe enums....

Reply:
done